### ISSUE TITLE
Icon shows incorrect numbers

### ISSUE THREAD

**Issue by vanowm** [Tampermonkey/tampermonkey#1848]
I couldn't find anything in the documentation what these numbers represent:
`2+2` or `2++` or `+0`

My guess was the first number represents number of **_running_** userscripts. Second number only shown when `Userscript Search integration` is set to `Add to icon badge text`, therefore I thought it represents number of **_available_** scripts for install...however it's not:

![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/31701f30-b254-4318-9223-a8bf1fbe729f)

Then I've noticed that the second number only shown when running userscript contains `require`:
```js
// ==UserScript==
// @name test script
// @namespace text 
// @version 1.0
// @author testing 
// @description test
// @match https://*/*
// @require file:///c:/test.js
// @noframes
// @grant none
// ==/UserScript==

console.log("Main test userscript", performance.now(), Math.random());
```
`c:/test.js` file:
```js
console.log("test.js file", performance.now(), Math.random());
```

Ok, so it counts running `require` files as a separate instance? Well, no, because adding more `require` or even pointing it to a non-existing file, doesn't increase or decrease the number. However, both number increase for each running instance of the userscript (aka for each iframe)

So, at this point, I'm totally lost what these numbers represent.

**Comment by vanowm** [comment]
Thanks for the explanation. However, I don't believe the issue is resolved here...
It shows wrong numbers
The second number shows number of running scripts, not number of found scripts:
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/5e6dda7c-6ec5-46f8-ab32-2ffe11b40739)
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/6445a7a3-7e4f-4d88-9059-346daba9af4c) ![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/a5d6f139-da7b-42aa-be25-9c4c1fa8375e)

If no scripts are running it shows `+0`:
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/beb8fa82-2cf7-4f29-9852-656a0f87f81d)

When no scripts found it doesn't show the number at all:
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/708ed937-597f-48ca-9cf5-fea3256dd39e)

If no scripts available but found some, it doesn't show anything:
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/3126f95d-6560-49b1-bd12-011501af8981)


When found scripts are in triple number, it shows `++` (which is the only case that works properly):
![image](https://github.com/Tampermonkey/tampermonkey/assets/511517/376189a2-b06c-4a3c-85e0-b4287292e2cc)

**Comment by derjanb** [comment]
This should be fixed at 5.0.6189 ([crx](https://data.tampermonkey.net/tampermonkey_beta.crx)|xpi in review)

Chrome/Edge users, please download the crx file linked above and drag and drop it to the extensions page `chrome://extensions`.
For a quick fix please export your settings and scripts as zip or (JSON) file at the "Utilities" tab and import it back at the fixed BETA version.
